code/datums/dna.dm

```diff
@@ -342,26 +505,41 @@
 /mob/living/carbon/proc/randmut(list/candidates, difficulty = 2)
 	if(!has_dna())
 		return
-	var/datum/mutation/human/num = pick(candidates)
-	. = num.force_give(src)
+	var/mutation = pick(candidates)
+	. = dna.add_mutation(mutation)
 
-/mob/living/carbon/proc/randmutb()
+/mob/living/carbon/proc/randmutb(scrambled = TRUE)
 	if(!has_dna())
 		return
-	var/datum/mutation/human/HM = pick((GLOB.bad_mutations | GLOB.not_good_mutations) - GLOB.mutations_list[RACEMUT])
-	. = HM.force_give(src)
-
-/mob/living/carbon/proc/randmutg()
-	if(!has_dna())
-		return
-	var/datum/mutation/human/HM = pick(GLOB.good_mutations)
-	. = HM.force_give(src)
-
-/mob/living/carbon/proc/randmutvg()

```

Comment:
very good helper disappeared ? Kinda unrelated but these helpers need to squashed into one + wrappers.

Reply:
It's just a persons native mutations now, so it'd be splitting up the few that they have even more.
I could also just make random mutations draw from the big pool. I'm not really sure wich is better